sections/04_study.md

```diff
@@ -36,9 +36,16 @@ from gene expression.  Those could be divided into separate sub-sections.*
 
 ### Transcription factors and RNA-binding proteins
 
-*Existing reviews have covered some of these papers rather well and we do not
-want to repeat what has already been well-stated elsewhere.  This could
-be split into two sub-sections or kept very brief.*
+Transcription Factor and RNA-binding proteins are key components for gene regulation, making them very important to understand for higher level biological processes. While high-throughput sequencing techniques such as ChIP-seq have been able to accurately identify binding regions for DNA and RNA proteins, these experiments are both time consuming and expensive. In addition, the sequencing methods do not provide any sort of analysis on the proteins which would lead to a better understanding of the underlying process. Thus, there is a need to computationally predict and understand these binding regions de novo from sequences.
```

Comment:
Hey, thanks a lot for your comments! Yeah, basically what I was trying to say is that even if sequence technologies were free, we still can't read DNA sequences as humans, so it's hard to interpret what they are doing (for other biological processes that depend on them).